domains/mcreed.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "M-creed",
+        "email": "marwankalosha005@gmail.com"
+    },
+    "record": {
+        "TXT": "4af6cbe09d82534a3e487fc8e6b54e"
```

Comment:
Is this a verification for some app?

Reply:
this is pages domain dns verification.
TXT record:
hostname: _github-pages-challenge-M-creed.mcreed.is-a.dev

value: 4af6cbe09d82534a3e487fc8e6b54e

Reply:
> this is pages domain dns verification. TXT record: hostname: _github-pages-challenge-M-creed.mcreed.is-a.dev
> 
> value: 4af6cbe09d82534a3e487fc8e6b54e

then you'd have to make _github-pages-challenge-M-creed.mcreed.json for that then

Reply:
do you mean that i make `_github-pages-challenge-M-creed.mcreed.json` in every repo i need to add this domain to it ?

Reply:
you'd need to make that for github pages verification